ReactWindows/ReactNative.Net46.Tests/UIManager/PropertySetterTests.cs

```diff
@@ -0,0 +1,486 @@
+﻿using NMock;
+using NUnit.Framework;
```

Comment:
I'm seeing the reference to NMock in a few files but not quite sure where it's being used.